jams/eval.py

```diff
@@ -54,10 +54,7 @@ def validate_annotation(ann, namespace):
         If `ann` fails schema validation
     '''
 
-    if not ann.search(namespace=namespace):
-        raise NamespaceError('Expected namespace="{:s}", found "{:s}"'
-                             .format(namespace, ann.namespace))
-
+    ann = convert(ann, namespace)
```

Comment:
Ok, I've updated the validation logic.

Instead of returning `true` (which was never actually checked) it now returns the coerced annotation, which is what gets piped into mir_eval.